examples/gallery/embellishments/pygmt_logo.py

```diff
@@ -0,0 +1,384 @@
+        # Save
+        # .............................................................................
+        # fig.show()
+        # fig_name = f"pygmt_logo_{shape}_{color_concept}_{bg_concept}"
+        fig_name = "pygmt_logo"
+        fig.savefig(fname=f"{fig_name}.eps")
+        # print(fig_name)
+
+        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        # Replot and apply rotation
+        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        fig = pygmt.Figure()
+        pygmt.config(MAP_FRAME_PEN="cyan@100")
+
+        bg_alpha = 100 if bg_transparent is True else 0
+        fig.basemap(
+            region=region,
+            projection=f"X{(size + 0.3) * 2}c",
+            frame=f"+g{color_bg}@{bg_alpha}",
+        )
+
+        fig.image(
+            imagefile=f"{fig_name}.eps",
+            position=f"jMC+w{size * 2}c",
+            # Rotation around z (vertical) axis placed in the center
+            perspective=f"{angle_rot}+w0/0",
+        )
+
+        # .............................................................................
+        # Save
+        # .............................................................................
+        # fig.show()
+        fig_name_rot = fig_name_logo = f"{fig_name}_rot{angle_rot}deg"
+        fig.savefig(fname=f"{fig_name_rot}.eps", resize=f"+m{margin}c")
+        # print(fig_name_rot)
+
+        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        # Replot and add wordmark "PyGMT"
+        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        if wordmark is True:
+            match orientation:
+                case "vertical":
+                    projection = f"X{size * 2 - 1.5}c/{size * 2}c"
+                    position = f"jTC+o0c/0.2c+w{size * 2 - 2.3}c"
+                    args_text = {"x": -3.2, "y": -2.8, "justify": "LM"}
+                    # args_cover = {"x": -2.2, "y": -2.8}
+                case "horizontal":
+                    projection = f"X{size * 2}c/{size - 2}c"
+                    position = f"jLM+o0.2c/0c+w{size - 2.3}c"
+                    args_text = {"x": -1.7, "y": 0, "justify": "LM"}
+                    # args_cover = {"x": -0.8, "y": 0}
+
+            fig = pygmt.Figure()
+            pygmt.config(MAP_FRAME_PEN="cyan@100")
+
+            bg_alpha = 100 if bg_transparent is True else 0
+            fig.basemap(
+                region=region, projection=projection, frame=f"+g{color_bg}@{bg_alpha}"
+            )
+
+            fig.image(imagefile=f"{fig_name_rot}.eps", position=position)
+
+            fig.text(text="PyGMT", font=f"45p,AvantGarde-Book,{color_gmt}", **args_text)
+            # fig.plot(style="s2.6c", fill=color_bg, **args_cover)
+            fig.text(text="Py", font=f"45p,AvantGarde-Book,{color_py}", **args_text)
```

Comment:
Use the `@;<color>;` syntax to change text color in a string (xref: https://docs.generic-mapping-tools.org/latest/reference/features.html#char-esc-seq)
```suggestion
            # fig.plot(style="s2.6c", fill=color_bg, **args_cover)
            fig.text(
                text=f"@;{color_py};Py@;;@;{color_gmt};GMT@;;", 
                font=f"45p,AvantGarde-Book", 
                **args_text
            )
```

Reply:
Yeah, I had in mind that there is such an option, but could not remember the exact syntax 🙃. But now updated in https://github.com/GenericMappingTools/pygmt/pull/3849/commits/470ed44390713dedc113d66d430debc351214199.